playwright/_impl/_js_handle.py

```diff
@@ -117,30 +117,38 @@ def serialize_value(value: Any, handles: List[JSHandle], depth: int) -> Any:
     if isinstance(value, str):
         return {"s": value}
 
+    if value in visited:
+        return dict(ref=visited.lookup_id(value))
+
     if isinstance(value, list):
-        result = list(map(lambda a: serialize_value(a, handles, depth + 1), value))
-        return dict(a=result)
+        visited[value] = True
```

Comment:
It looks like there are two different pattern for StrongDict use: in impl_to_api and here.

Reply:
Yes, here's we're just using it essentially as a Set (but built in `set(…)` requires hash impl)